test-it/karma/fixtures/events/ValueChangeStaticControlEventNullJavaScriptRefNum.via

```diff
@@ -0,0 +1,55 @@
+define (UpdateBooleanOnValueChangeEvent dv(.VirtualInstrument (
+    Events: c(
+        e(c( // Event Struct 0
+           e(c(  // Event spec 0
+               e(dv(UInt32 1000) eventSource)  // event source enum
+               e(dv(UInt32 2) eventType)  // ValueChanged
+               e(dv(UInt32 18) controlUID)  // for static control refs
```

Comment:
This should be a JavaScriptRefnum, not a controlUID.

Maybe we should actually enforce this, and check the type when you use ConfigureEventSpecjSRef.

Reply:
This won't really be related to ConfigureEventSpecJSRef, since this information isn't actually fed into ConfigureEventSpecJSRef. We need the control ID in the event spec at some point, because otherwise we won't know what the ID actually is, since JavaScriptRefNums are opaque to Vireo.